### PR TITLE
WIP: added UserProfile and moved py src into /cafebabel

### PR DESCRIPTION
**Pull request by vinyll**

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ipdb
+ipython
```
Comment (vinyll) [inline requirements-dev.txt:3]: I feel like ipython and ipdb are more personal choices, or do we actually use them in shared code somehow?

Comment (davidbgk) [reply requirements-dev.txt:3]: I don't have hard feelings on what is happening in -dev but I always need those :)

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# The reason we use deb file directly from circle-ci is speed:
+# https://discuss.circleci.com/t/speed-up-python-3-5-1-builds/1760/7
+machine:
+  pre:
+    - wget https://s3.amazonaws.com/circle-downloads/circle-pyenv-python-3.5.1_1.0_amd64.deb
+    - sudo dpkg -i circle-pyenv-python-3.5.1_1.0_amd64.deb
+  python:
+    version: 3.5.1
```
Comment (vinyll) [inline circle.yml:8]: https://stackoverflow.com/a/44318670/328117

File: cli.py [hunk 3]
```diff
@@ -15,6 +15,12 @@ def _initdb():
     click.echo('DB intialized.')
 
 
+def _dropdb():
+    tables = [User, UserProfile, Role, UserRoles]
+    drop_model_tables(tables, fail_silently=True)
+    click.echo('DB droped.')
```
Comment (vinyll) [inline cli.py]: _dropped_

**Pull request merged**